pkg/util/k8sutil/k8sutil.go

```diff
@@ -346,7 +346,7 @@ func newEtcdPod(m *etcdutil.Member, initialCluster []string, clusterName, state,
 		},
 		Spec: v1.PodSpec{
 			InitContainers: []v1.Container{{
-				Image: "busybox:1.28.0",
+				Image: "busybox:1.28.0-glibc",
```

Comment:
Can you add a comment about the search path issue on VMWare environment and why use glibc instead?